searx/settings.yml

```diff
@@ -2037,6 +2037,15 @@ engines:
     categories: videos
     disabled: true
 
+  - name: livespace
+    engine: livespace
+    shortcut: ls
```

Comment:
Sus shortcut :eyes:

Reply:
true, how much better is livesp, or do you have a good suggestion?

Reply:
No, it's fine, I just find it funny because
```shell
$ ls -al
-rwxrw-r--    10    root   root 2048    Feb 18 22:01 foo.bar
```
is what `ls` means to me :)

Reply:
there's also `bin` :p (I digress)